schema.sql

```diff
@@ -0,0 +1,40 @@
+CREATE DATABASE doingsdone_db
+    DEFAULT CHARACTER SET utf8
+    DEFAULT COLLATE utf8_general_ci;
+
+USE doingsdone_db;
+
+CREATE TABLE users (
+    email VARCHAR(100) NOT NULL,
+    CREATE UNIQUE INDEX uk_email (email),
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(128) NOT NULL UNIQUE,
```

Comment:
name wouldn't be unique. now you forbid, for example, sign up couple users with name Alexander Ivanov. just email unique, see SRS